taigit/src/libraries/Taiga.ts

```diff
@@ -64,14 +78,31 @@ interface spr_stats {
  * @returns boolean dictating success of login
  */
 export async function
-taiga_login(username : string, password : string) : Promise<boolean> {
+taiga_login(username : string, password : string) : Promise<login_response> {
     let response = await axios.post("https://api.taiga.io/api/v1/auth", {
         "password": password,
         "type": "normal",
         "username": username
-    })
+    });
+
+    let data = response.data;
+    return {
+        success : (response.status == 200),
```

Comment:
Does taiga provide an error message on unsuccessful logins that we could return for the frontend to show?

Reply:
The api only specifies the return code of 200 on successful login:
https://taigaio.github.io/taiga-doc/dist/api.html#auth-normal-login

If you want to differentiate types of login failures I could return HTTP status text too but it's not much more descriptive: https://www.restapitutorial.com/httpstatuscodes.html

I think it would be fine to just display "Login Failed" when the login fails, no matter the reason.